zanata-war/src/main/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -293,6 +293,15 @@
     <to last="true">/webtrans/Application.seam$1</to>
   </rule>
 
+  <!-- zanata-assets resource -->
+  <!-- JSF serves zanata-assets with suffix of .seam,
+      this is to make sure any reference to zanata-assets without .seam can access to the resource.
+      e.g. /assets/style.css are redirect to /assets/style.css.seam -->
```

Comment:
How about this?

    eg a request for /assets/style.css will be redirected to /assets/style.css.seam